daemon/cmd/daemon_main.go

```diff
@@ -1589,7 +1590,7 @@ func newDaemonPromise(params daemonParams) promise.Promise[*Daemon] {
 			return nil
 		},
 	})
-	return daemonPromise
+	return daemonPromise, struct{}{}
```

Comment:
if `DaemonInitialization` has type `struct{}` then we can do `legacy.DaemonInitialization{}` here.

Reply:
yep, changed